code/src/pages/moviesDetails.css

```diff
@@ -0,0 +1,135 @@
+.movie {
+  display: flex;
+  background-size: cover;
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  width: 100%;
+}
+.movie-details {
+  display: flex;
+  align-items: flex-end;
+  padding: 20px;
+  color: white;
+}
+.movie-poster {
+  display: flex;
+  width: 50%;
+  border: 5px solid #fff;
+  width: 400px;
+  margin: 50px;
+}
+.details-container {
+  margin-bottom: 50px;
+}
+.details-container h2 {
+  font-size: 22px;
+}
+.back-arrow {
+  transform: translate(-40%, 29%);
+  width: 30px;
+  height: 30px;
+}
+span.movie-rating {
+  font-size: 20px;
+  color: red;
+  font-weight: 400;
+  margin-left: 10px;
+}
+p.movie-overview {
+  width: 45%;
+}
+h3 {
+  justify-content: center;
+  text-align: center;
+  top: 50;
+  font-size: 20px;
+}
+.back-link:hover,
+.back-link:focus,
+.back-link:active {
+  -webkit-transform: translateX(8px);
+  transform: translateX(8px);
+  color: aquamarine;
+}
+.back-link {
+  position: absolute;
+  color: #fff;
+  align-items: center;
+  font-weight: 600;
+  margin: 50px;
+  padding-left: 30px;
+  text-decoration: none;
+  font-size: 18px;
+  -webkit-transform: perspective(1px) translateZ(0);
+  transform: perspective(1px) translateZ(0);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+  -webkit-transition-duration: 0.3s;
+  transition-duration: 0.3s;
+  -webkit-transition-property: transform;
+  transition-property: transform;
+}
+
+/* MEDIA QUERIES */
+@media (min-width: 280px) and (max-width: 668px) {
+  .movie-poster {
+    width: 100%;
+    border: 3px solid #fff;
+    width: 300px;
+    margin: 15px 15px -50px;
+    margin-top: 50px;
+  }
+  p.movie-overview {
+    width: 80%;
+    margin: 15px 15px -50px;
+  }
+  .movie-details {
+    flex-wrap: wrap;
+    margin: 15px 15px 0;
+    padding: 30px;
+    margin: auto;
+  }
+  .details-container h2 {
+    padding-top: 50px;
+    margin: 20px 15px;
+  }
+  a.back-link {
+    margin: 30px;
+  }
+}
+@media (min-width: 669px) and (max-width: 1024px) {
```

Comment:
I think there's something not right here, when i watch on my computer, the 'posters' get super small and splits up?